WordPress/gutenberg
# DataViews grid layout: Fix long pattern names display

## What?

Fixes https://github.com/WordPress/gutenberg/issues/65134

## Why?

Long pattern names are cut off when the grid layout is used

## Testing Instructions

- Go to site editor
- Open pattern list
- Check pattern with long names

## Screenshots or screencast 

<img width="1064" alt="image" src="https://github.com/user-attachments/assets/b7450438-27c2-42fa-8e03-4dafe0de315e">
